src/style.rs

```diff
@@ -33,62 +36,72 @@ impl Color {
             Color::Blue => "b",
             Color::Magenta => "m",
             Color::Cyan => "c",
-            Color::Gray => "g",
-            Color::DarkGray => "G",
+            Color::Gray => "w",
+            Color::DarkGray => "B",
```

Comment:
For the record, I made these changes to aid debugging, since the codes were ambiguous before (`G` = light green and gray) and additionally "gray" is usually called "white", "dark gray" usually called "light black" and "white" usually called "light white" in other terminals.

Reply:
That is totally fine with me. You are right that the existing debug codes were not well thought in the first place.